src/generateRoutes.js

```diff
@@ -61,17 +75,112 @@ function generateRoutes(routes, app, endpointInfo = false) {
     }
 
     if (route.delete) {
-      const endpoint = `/${route.name}/:${route.delete.id || idFieldName || 'id'}`
-      info(`    DELETE ${endpoint} -> ${route.delete.usecase().description}`)
-      app.delete(endpoint, async (req, res, next) => {
+      const path = forcedPath.delete?.path || generatedPath.delete
+      const verb = forcedPath.delete?.verb || 'delete'
+      info(`    ${verb.toUpperCase()} ${path} -> ${route.delete.usecase().description}`)
+      app[verb](path, async (req, res, next) => {
         const request = { params: req.params }
         const usecase = route.delete.usecase
         const currentController = route.delete.controller
 
         await currentController(usecase, request, req.user, res, next)
       })
     }
+
+    if (route.other) route.other.forEach((other) => {
+      let path
+      let resourceName
+      const verb = getVerb(other?.REST) || 'post'
+      if (other.REST?.resourceName) {
+        resourceName = `/${other.REST?.resourceName}`
+      }
+      if (other.REST) {
+        path = other.REST[verb] || resourceName
+      }
+      if (path) {
+        info(`    ${verb.toUpperCase()} ${path} -> ${other.usecase().description}`)
+        app[verb](path, async (req, res, next) => {
+          const request = { body: req.body, query: req.query, params: req.params }
+          const usecase = other.usecase
+          const currentController = other.controller
+
+          await currentController(usecase, request, req.user, res, next)
+        })
+      }
+    })
   })
 }
 
+function getForcedPath(route) {
+  const forcedPaths = {}
+
+  const verbs = {
+    getAll: getVerb(route.getAll?.REST),
+    getById: getVerb(route.getById?.REST),
+    post: getVerb(route.post?.REST),
+    put: getVerb(route.put?.REST),
+    delete: getVerb(route.delete?.REST)
+  }
+
+  if (verbs.getAll) {
```

Comment:
Those "if's" could be refactored since all do the same just changing the verb used.